### ISSUE TITLE
Bump `release-please` version to 13.6.0 ?

### ISSUE THREAD

**Issue by erikmd** [google-github-actions/release-please-action#453]
### TL;DR

Could you please make a new release of the GH action so one could benefit from the latest `release-please` features?

→ including PR https://github.com/googleapis/release-please/pull/1325 that was shipped in [13.5.0](https://github.com/googleapis/release-please/releases/tag/v13.5.0)

Cc @chingor13 @bcoe FYI

### Observed behavior

The GH action currently uses:

https://github.com/google-github-actions/release-please-action/blob/98d3822b5f1a4fd9ef0bdd1fdf33bc79acd4a328/package.json#L29

### Action YAML

```yaml
"N/A"
```

### Additional information

This issue is just a follow-up of this earlier comment: https://github.com/googleapis/release-please/pull/1328#issuecomment-1063251461

**Comment by bcoe** [comment]
done 👍

**Comment by erikmd** [comment]
Thanks :tada:
